doc/source/guide_lenses.rst

```diff
@@ -0,0 +1,117 @@
+Guide to Lenses
+===============
+
+Overview
+--------
+
+A Lens can be either a zone-app or a service.
+They are modular and comprehensive units that help to focus on smaller sections of the workspace.
+
+Functions
+---------
+
+Get
+***
+
+Get all Lenses. Can be filtered based on enforcement and starred status.
+
+.. tabs::
+   .. code-tab:: bash Command Line
+
+         ./araalictl api -fetch-enforcement-status
+         # "-enforced" tag optional
+         ./araalictl api -fetch-starred-lens
+
+   .. code-tab:: py
+
+         api.Lens.get(enforced, starred)
+         # enforced: default=False
```

Comment:
both parameters are optional: might be better to say it as:
api.Lens.get(enforced=True, starred=True)